.goreleaser.yml

```diff
@@ -1,4 +1,7 @@
 project_name: cloud-platform-cli
+env:
+  - CGO_ENABLED=0
```

Comment:
I think we can get rid of this now, tbh.

Reply:
removed